roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -1,6 +1,12 @@
 ---
 - include_tasks: pre_install.yaml
 
+- name: Set metrics project
+  oc_project:
```

Comment:
when creating the project as part of migrating from openshift-infra, the following annotations must be copied from openshift-infra:

```
openshift.io/sa.scc.mcs
openshift.io/sa.scc.supplemental-groups
openshift.io/sa.scc.uid-range
```

that ensures the same uid/selinux/supplemental groups are defaulted/forced onto the metrics pods as in the old namespace